### ISSUE TITLE
Add NuGet to Maui Project

### ISSUE THREAD

**Issue by criistii** [redth-org/BTProgressHUD#101]
Hey,
I am having issues adding the 2.0 nuget to a new Maui project in Visual Studio for Mac. Any idea what is going on?

**Comment by Cheesebaron** [comment]
You need to provide more details.

**Comment by criistii** [comment]
Sorry for the delay. Steps to reproduce: on MacOS Ventura > VS 17.5.4 (build 8) > Create a new Maui project from the Terminal > Open it > Manage Nuggets for the iOS Platform> Find & Add BTProgressHUD > "Could not add BTProgressHUD"

Tried to add it to iOS only and still got the following error:
Package BTProgressHUD 2.0.0 is not compatible with net7.0-android33.0

**Comment by Cheesebaron** [comment]
Yes, BTProgressHUD is not compatible with Android. I don't know why your package manager is trying to add it to the Android project, but that is not a supported use case.

You may have to add it manually to your csproj file only for your iOS target.

**Comment by criistii** [comment]
Just tried adding it in the csproj using 
`<PackageReference Include="BTProgressHUD" Version="2.0.0" Condition="'$(TargetFramework)' == 'net7.0-ios'" />`
but it did not work. By any chance, you have a working example?
Would be great if this info was in the documentation.

**Comment by Cheesebaron** [comment]
You will probably have to have the condition on an ItemGroup:

```xml
<ItemGroup Condition=" $(TargetFramework.StartsWith('net7.0-ios')) ">
    <PackageReference Include="BTProgressHUD" Version="2.0.0" />
</ItemGroup>
```

**Comment by criistii** [comment]
Sadly, it's not working. Probably a VS issue.

**Comment by Cheesebaron** [comment]
I created a blank MAUI project, added: 

```
<ItemGroup Condition=" $(TargetFramework.StartsWith('net7.0-ios')) ">
    <PackageReference Include="BTProgressHUD" Version="2.0.0" />
</ItemGroup>
```

Works fine when I run: `dotnet restore`

What is the error you are seeing?
